tests/lib/rules/jest/no-vague-titles.js

```diff
@@ -125,6 +125,18 @@ ruleTester.run('no-vague-titles', rule, {
       code: `test.each([['production'], ['staging']])('Includes things for %s clients')`,
       parser,
     },
+    {
+      code: `import('./foo')`,
+      parser,
+    },
+    {
+      code: 'foo(bar)()',
+      parser,
+    },
+    {
+      code: 'super()',
```

Comment:
Ehhhhh, I'd really like to see this used in a meaningful context.

Reply:
@GoodForOneFare do you have any suggestions for this one? or should we leave it as is and deal with the input file filter later?

Reply:
This should work:
```
    {
      code: 'class Foo { constructor() { super(); } }',
      parser: 'babel-eslint',
    },
```

Reply:
I swear i tried something like that. clearly i was imagining it 😁